src/main/resources/static/js/subscription.js

```diff
@@ -519,6 +547,38 @@ jQuery(document).ready(function () {
     // /Stop ## Reload Table#################################################
 
     // /Start ## Bulk delete#################################################
+    function deleteSubscription(subscriptionsToDeleteString) {
```

Comment:
I guess one or several subscriptions can be deleted with this function,, then its plural and the function name should be in plural with a "s" character in the end of the name ->  "deleteSubscriptions(...)"